packages/server/lib/server.ts

```diff
@@ -39,7 +39,6 @@ if (NANGO_MIGRATE_AT_START === 'true') {
     logger.info('Not migrating database');
 }
 
-await environmentService.cacheSecrets();
```

Comment:
I imagine caching secrets was done for a good reason. Is this reason no longer relevant?

Reply:
As stated in the PR desc, caching would now cache not just the ID but the full objects. I'm not entirely confident to do that because of everything that could modify those two objects. I feel like it was done before all our performance improvements, and specifically because secretKey was encrypted (we would have to pull everything from the DB to find the unencrypted key). But now that we query by hash it's not really relevant. 
I do wonder about the added ms, I'm not worrying about API it has fairly low usage, but for persist it might be more costly.


I'll let @khaliqgant complete with more context